sys/parsoid.js

```diff
@@ -173,6 +173,9 @@ function stripAcceptFromVary(res) {
         .filter(header => !/^accept$/i.test(header))
```

Comment:
Let's add a check for falsy element here for more safety?

Reply:
Done.